config/ember-try.js

```diff
@@ -7,7 +7,7 @@ module.exports = function() {
     getChannelURL('release'),
     getChannelURL('beta'),
     getChannelURL('canary')
-  ]).then((urls) => {
+  ]).then((/* urls */) => {
```

Comment:
This was an unused variable. source channel URLs are being retrieved, but are not used. I'll take care of this in a subsequent PR